tictactoe-domain/src/main/java/camp/nextstep/edu/tictactoe/domain/strategy/IntermediateStrategy.kt

```diff
@@ -0,0 +1,17 @@
+package camp.nextstep.edu.tictactoe.domain.strategy
+
+import camp.nextstep.edu.tictactoe.domain.CellPosition
+import camp.nextstep.edu.tictactoe.domain.Owner
+import camp.nextstep.edu.tictactoe.domain.TictactoeMap
+import camp.nextstep.edu.tictactoe.domain.WinnerChecker
+
+internal class IntermediateStrategy : RandomStrategy() {
+    override fun getNextTurnPosition(tictactoeMap: TictactoeMap): CellPosition {
+        val position = WinnerChecker.getPlayerWinPosition(tictactoeMap.positions, Owner.X)
+        if (position != null) {
+            return position
+        }
+
+        return super.getNextTurnPosition(tictactoeMap)
```

Comment:
```suggestion
        return WinnerChecker.getPlayerWinPosition(tictactoeMap.positions, Owner.X)
            ?: super.getNextTurnPosition(tictactoeMap)
```
좀 더 간결하게 표현할수도 있을거같아요 😄

Reply:
네 말씀하신 것처럼 로직 수정하였습니다.